File: 1266. Minimum Time Visiting All Points/solution.py
class Solution(object):
  def minTimeToVisitAllPoints(self, points):
    """
    :type points: List[List[int]]
    :rtype: int
    """
    distance = 0
    for index in range(1, len(points)):
      last = points[index - 1]
      current = points[index]
      most = abs(current[0] - last[0])
      if abs(current[1] - last[1]) > most:
        most = abs(current[1] - last[1])
      distance += most
    return distance

File: 1266. Minimum Time Visiting All Points/solution.cpp
class Solution {
public:
  int minTimeToVisitAllPoints(vector<vector<int>>& points) {
    int distance = 0;
    for (int index = 1; index < points.size(); index++){
      int most = abs(points[index][0] - points[index - 1][0]);
      if (abs(points[index][1] - points[index - 1][1]) > most){
        most = abs(points[index][1] - points[index - 1][1]);
      }
      distance += most;
    }
    return distance;
  }
};